RevenueCat/Scripts/StoreProduct.cs

```diff
@@ -10,23 +10,83 @@ public partial class Purchases
     /// </summary>
     public class StoreProduct
     {
+        /// <summary>
+        /// Title of the product.
+        /// </summary>
+        /// <returns></returns>
         public readonly string Title;
+
+        /// <summary>
+        /// Product Id.
+        /// </summary>
+        /// <returns></returns>
         public readonly string Identifier;
+
+        /// <summary>
+        /// Description of the product.
+        /// </summary>
+        /// <returns></returns> 
         public readonly string Description;
+
+        /// <summary>
+        /// Price of the product in the local currency.
+        /// Contains the price value of DefaultOption for Google Play.
+        /// </summary>
+        /// <returns></returns>
         public readonly float Price;
+
+        /// <summary>
+        /// Formatted price of the item, including its currency sign.
+        /// Contains the formatted price value of DefaultOption for Google Play.
+        /// </summary>
+        /// <returns></returns>
         public readonly string PriceString;
+
+        /// <summary>
+        /// Currency code for price and original price.
+        /// Contains the currency code of DefaultOption for Google Play.
+        /// </summary>
+        /// <returns></returns>
         [CanBeNull] public readonly string CurrencyCode;
+        
+        /// <summary>
+        /// Introductory price of the product. Null if no introductory price is available.
+        /// It contains the free trial if available and user is eligible for it.
+        /// Otherwise, it contains the introductory price of the product if the user is eligible for it.
+        /// This will be null for non-subscription products.
+        /// </summary>
+        /// <returns></returns>
```

Comment:
This docs will make sense after https://github.com/RevenueCat/purchases-hybrid-common/pull/952